app/helpers/phases_helper.rb

```diff
@@ -9,4 +9,26 @@ def phase_number(challenge, phase)
   def phase_has_recused_evaluator?(phase)
     phase.evaluator_submission_assignments.recused.exists?
   end
+
+  def statuses
+    {
+      draft: "Draft",
+      gsa_review: "GSA review",
+      approved: "Approved",
+      edits_requested: "Edits requested",
+      unpublished: "Unpublished",
+      published: "Published",
+      archived: "Archived"
+    }
+  end
+
+  def evaluation_status(phase)
+    if phase.submissions.eligible_for_evaluation.where.not(status: [:in_progress, :completed]).exists?
+      "Not Started"
+    elsif phase.submissions.eligible_for_evaluation.where.not(status: [:not_started, :in_progress]).exists?
+      "Completed"
+    else
+      "In Progress"
+    end
```

Comment:
this should be `submissions.evaluation_status`
```suggestion
    if phase.submissions.eligible_for_evaluation.where.not(evaluation_status: [:in_progress, :completed]).exists?
      "Not Started"
    elsif phase.submissions.eligible_for_evaluation.where.not(evaluation_status: [:not_started, :in_progress]).exists?
      "Completed"
    else
      "In Progress"
    end
```

Reply:
can you add the colored status labels to these? 
<img width="125" alt="Screenshot 2025-02-18 at 12 31 48 PM" src="https://github.com/user-attachments/assets/592d70f7-7d5a-4cc3-bd45-fce58d330538" />

Reply:
might be something like this:
```
<span class="text-top usa-tag <%= evaluation_submission_assignment_status_color(submission) %>"><%= submission.evaluation_status.to_s.titleize %></span>
```